packages/crypto/Cargo.toml

```diff
@@ -9,24 +9,41 @@ license = "Apache-2.0"
 
 [features]
 default = []
-std = ["dep:thiserror"]
+std = [
+    "dep:ark-bls12-381",
+    "dep:ark-ec",
+    "dep:ark-ff",
+    "dep:ark-serialize",
+    "dep:num-traits",
+    "dep:rayon",
+    "dep:sha2",
+    "dep:thiserror"
+]
```

Comment:
We need the `no_std` version to access the `cosmwasm-crypto` error type inside `cosmwasm-core`, since we had to move all the error types into core to keep the API the same.

If `cosmwasm-crypto` would depend on std, `cosmwasm-core` wouldn't build on `no_std` targets anymore.